### DISCUSSION TITLE
DIRACScript class is deprecated

### DISCUSSION THREAD

**Discussion by arrabito** [DIRACGrid/DIRAC#6774] [Q&A] [answered]
Dear all,

Some time ago, we have updated our DIRAC extension to use:

`DIRAC.Core.Utilities.DIRACScript`

instead of:

`DIRAC.Core.Base.Script `

Now with Version: 8.0.5 we get this warning:

`NOTE: DIRACScript class is deprecated (DIRACScript is deprecated, use 'from DIRAC.Core.Base.Script import Script' instead.).`

Does it mean that we should revert back using:

`DIRAC.Core.Base.Script `

?

Thank you.

**Comment by fstagni** [comment, marked as answer]
There have been a bit of back-and-forth, but from v8.0 `DIRACScript` is deprecated (still working thought): https://github.com/DIRACGrid/DIRAC/wiki/DIRAC-8.0#deprecated

**Comment by arrabito** [comment]
OK thank you no problem, we will change our imports.
